Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -161,46 +220,132 @@ For the example of the _'smart attentive notifications'_, the following happens:
+All you need is to trigger the rotation of the holographic gem from the _EyeTrackingTarget_'s _WhileLookingAtTarget()_ event. 
+Here are a few more details:
+
+1. Create a generic script that includes a public function to rotate the GameObject it is attached to. 
+Below is an example from _RotateWithConstSpeedDir.cs_ where we can tweak the rotation direction and speed from the Unity Editor. 
+
+```csharp 
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking
+{
+    /// <summary>
+    /// The associated GameObject will rotate when RotateTarget() is called based on a given direction and speed.
+    /// </summary>
+    public class RotateWithConstSpeedDir : MonoBehaviour
+    {
+        [Tooltip("Euler angles by which the object should be rotated by.")]
+        [SerializeField]
+        private Vector3 RotateByEulerAngles = Vector3.zero;
+
+        [Tooltip("Rotation speed factor.")]
+        [SerializeField]
+        private float speed = 1f;
+
+        /// <summary>
+        /// Rotate game object based on specified rotation speed and Euler angles.
+        /// </summary>
+        public void RotateTarget()
+        {
+            transform.eulerAngles = transform.eulerAngles + RotateByEulerAngles * speed;
+        }
+    }
+}
+```
+
+2. Add the [EyeTrackingTarget](xref:Microsoft.MixedReality.Toolkit.Input.EyeTrackingTarget) script to your target GameObject and reference the _RotateTarget()_ function in the UnityEvent trigger as shown the screenshot below:
+
+![EyeTrackingTarget sample](../Images/EyeTracking/mrtk_et_EyeTrackingTargetSample.jpg)
+
+
+
+
+### Example #3: Pop those gems aka _multimodal eye-gaze-supported target selection_
+In the previous example, we have shown how easy it is to detect whether a target is looked at and how to trigger a reaction to that.
+Next, let's make the gems explode using the _OnSelected()_ event from the [EyeTrackingTarget](xref:Microsoft.MixedReality.Toolkit.Input.EyeTrackingTarget). 
 The interesting part is _how_ the selection is triggered. 
 The [EyeTrackingTarget](xref:Microsoft.MixedReality.Toolkit.Input.EyeTrackingTarget)
 allows for quickly assigning different ways to invoke a selection:
 
-- _Pinch gesture_: Setting the 'Select Action' to 'Select' uses the default hand gesture to trigger the selection. This means that the user can simply raise their hand and pinch their thumb and index finger together to confirm the selection.
+- _Pinch gesture_: Setting the 'Select Action' to 'Select' uses the default hand gesture to trigger the selection. 
```

Comment:
Do you know if this is only possible with the ET examples because of how they disable the far rays?

Reply:
I think my followup question here is around providing a section that explains to users how they modified the default profile or HL2 profile to remove the far interactions that would allow for this to work, and then to provide guidance on when this might be appropriate (or when it wouldn't be).

i.e. I think this involves deleting the default controller from the articulated hand controller type, and if so, it might be good to explicitly show people how to do this.

Reply:
Yeah, I'm still trying to figure out what the best flow for explaining it all is. But yep... we should definitely add some info on how to set up the profiles to make it work as expected.